<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConvertImagesDialog</class>
 <widget class="QDialog" name="ConvertImagesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>96</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>96</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convert Images</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(45, 45, 45);
color: #fff;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name Suffix:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_nameSuffix">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(88, 88, 88);
color: rgb(255, 255, 255);

border-style: inset;
border-width: 1px;
border-color: rgb(74, 127, 227);
border-radius: 3px;</string>
       </property>
       <property name="text">
        <string>_converted</string>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_format">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
background-color: rgb(88, 88, 88);
color: rgb(255, 255, 255);

border-style: inset;
border-width: 1px;
border-color: rgb(74, 127, 227);
border-radius: 3px;
}

QComboBox::down-arrow {
	color: rgb(158, 158, 158, 0);
}</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>.jpg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>.png</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>.tif</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>.ppm</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>.bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>.xpm</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_jpgQuality">
       <property name="text">
        <string>JPG Quality:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_jpgQuality">
       <property name="styleSheet">
        <string notr="true">QSpinBox {
	border-style: inset;
border-width: 1px;
border-color: rgb(74, 127, 227);
border-radius: 3px;
 }

QSpinBox::up-arrow {
    color: rgb(180, 180, 180);
 }

QSpinBox::down-arrow {
    color: rgb(180, 180, 180);
 }</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>98</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_convert">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(88, 88, 88);

border: 1px solid;
border-color: rgb(74, 127, 227);
border-radius: 3px;

height: 16px;</string>
       </property>
       <property name="text">
        <string>Convert</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="styleSheet">
      <string notr="true">QProgressBar {
    border: 1px solid  rgb(74, 127, 227);
    border-radius: 4px;
	text-align: center;
 }

 QProgressBar::chunk {
	
	background-color: rgb(51, 96, 191);
    width: 20px;
 }</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
